doc/design/linc_permissions.md

```diff
@@ -0,0 +1,52 @@
+# Permissions for S3 Assets in LINC Archive
+
+Collaborators: Aaron Kanzer, Kabilar Gunalan
+
+## Requirements
+
+• As a CONNECTS developer, I shall be able to view multiple datasets/layers (e.g. dMRI, tractography, HiP-CT) in the same neuroglancer
+• As a CONNECTS developer, I shall be able to view data locally or on neuroglancer
+• As a CONNECTS developer, I shall be able to have direct access to S3 asset URI
+• As a LINC Archive user, I shall be able to render neuroglancer layers view similar to DANDI
```

Comment:
```suggestion
4. As a LINC user, use the `External Services` button in the `FileBrowser` to load a Neuroglancer viewer with a single layer of an asset.  This requirement mirrors the capabilities in the DANDI Archive.
```

Reply:
This requirement may not be necessary if we proceed with WEBKNOSSOS as they would provide a mechanism for LINC users to authenticate and visualize the data.

Reply:
As per our recent discussion, I am moving this to the first use case to address.